cmd/explorer/main.go

```diff
@@ -377,102 +377,103 @@ func main() {
 			)
 
 			router.HandleFunc("/", handlers.Index).Methods("GET")
-			router.HandleFunc("/latestState", handlers.LatestState).Methods("GET")
-			router.HandleFunc("/launchMetrics", handlers.SlotVizMetrics).Methods("GET")
+			router.HandleFunc("/turnstile/verify", handlers.VerifyTurnstile).Methods("GET")
```

Comment:
Also I'm getting a couple errors on the index page when I'm on your branch. Clicked I am human and these errors are piling up, they do not appear on the master branch:
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/5877862/3bb6617a-fbf6-40b6-b095-1a9e2b3a861c)
You can ignore the ads errors, probably due to adblock.

Reply:
I think my branch was behind, I merged the current master an this seems to be resolved now